sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -310,6 +317,9 @@ async def create_container_if_not_exists(
         :keyword int analytical_storage_ttl: Analytical store time to live (TTL) for items in the container.  A value of
             None leaves analytical storage off and a value of -1 turns analytical storage on with no TTL. Please
             note that analytical storage can only be enabled on Synapse Link enabled accounts.
+        :keyword Dict[str, Any] vector_embedding_policy: **provisional** The vector embedding policy for the container.
```

Comment:
I think we need to update the typehint for `indexing_policy` to `Dict[str, Any]` throughout.